### ISSUE TITLE
payload 2.4 and slate 1.3 does conflict on type

### ISSUE THREAD

**Issue by dominikzogg** [payloadcms/payload#4406]
### Describe the Bug

After the upgrade to payload 2.4 i could not built the app (dev server runs fine) i get type errors on the editor field (main config and field config)

```
src/payload.config.ts:106:3 - error TS2322: Type 'RichTextAdapter<any[], AdapterArguments, AdapterArguments>' is not assignable to type 'RichTextAdapter<any, any, any>'.
  Type 'RichTextAdapterBase<any[], AdapterArguments, AdapterArguments> & { CellComponent: FC<CellComponentProps<RichTextField<any[], AdapterArguments, AdapterArguments>>>; FieldComponent: FC<...>; }' is not assignable to type 'RichTextAdapter<any, any, any>'.
    Type 'RichTextAdapterBase<any[], AdapterArguments, AdapterArguments> & { CellComponent: FC<CellComponentProps<RichTextField<any[], AdapterArguments, AdapterArguments>>>; FieldComponent: FC<...>; }' is not assignable to type 'RichTextAdapterBase<any, any, any> & { CellComponent: FC<CellComponentProps<any>>; FieldComponent: FC<RichTextFieldProps<any, any, any>>; }'.

106   editor: slateEditor({}),
      ~~~~~~

  node_modules/payload/dist/config/types.d.ts:466:5
    466     editor: RichTextAdapter<any, any, any>;
            ~~~~~~
    The expected type comes from property 'editor' which is declared here on type 'Config'
   ```

### To Reproduce

use playload 2.4 with slate 1.3

### Payload Version

2.4.0

**Comment by dominikzogg** [comment]
the error is only visible if the project was built with the NextJS/Payload combo template (so it depends on how strict the TS settings are)

**Comment by AlessioGr** [comment]
Hey @dominikzogg this should now be fixed in richtext-slate 1.3.1 :)

**Comment by dominikzogg** [comment]
@AlessioGr tested, works fine, thank you
